.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -2,151 +2,254 @@
-      - [Internal] Don't merge the PR.
-1.  - [ ] Coordinate with team and complete manual distributed smoke-testing. Suggested assignment below, but it's flexible.  Send email with direct links to artifacts from green PR.
-      - [2.1]
-        - [ ] RHEL7 and RHEL8 VMs - crummel
-        - [ ] RHEL docker - dleeapho
-        - [ ] Fedora30 - dseefeld
-        - [ ] Fedora31 - dagood
-        - [ ] Debian9 - crummel
-      - [3.1]
-        - [ ] RHEL 7 & 8 VM - crummel
-        - [ ] RHEL Docker - dleeapho
-        - [ ] Fedora30 - dseefeld
-        - [ ] CentOS 7 - dagood
-        - [ ] CentOS 8 - dseefeld
-        - [ ] Debian - dagood
-1.  - [ ] [Internal] Download the artifacts from the green PR validation build onto a machine with Bash.
-      - The smoke-test prereqs are platform-independent but should be taken from a non-portable build, e.g. CentOS7 Offline, *not* CentOS7 Offline Portable.
-      - We may require only a subset of the artifacts for certain releases: refer to the previous release email thread corresponding to 2.1 vs. 3.1 build.
-      - Can use <https://github.com/dotnet/source-build/blob/release/3.1/scripts/fetch-azdo-artifact-tarball.sh> as a library.
-        - E.g. download the tarballs with `download_tarball`, and fix them up with `fix_azdo_tarball` if necessary.
-1.  - [ ] [Internal] Rename tarball and smoke-test prereqs to human-friendly names. (Can use `rename_tarball_inner_dir`.)
+        - If the baseline change is due to the change in patch version, make
+          sure there's an issue tracking fixing the instability, or start
+          tracking it.
+        - If in doubt, talk to the team. Someone might have already encountered
+          a similar diff in the past.  The Source-Build Teams channel is a good
+          place to start the conversation, or the dotnetADTeam DL.
+1. - [ ] Add team members crummel, LoganBussell, and msimons as reviewers to
+         the PR. Get acceptance.
+      - Note: Don't merge the PR.  This is just used to generate the pre-embargo-date
+        tarball for our partners.  This will be ported to the public repo
+        later and become the official release.
+1. - [ ] Download the artifacts from the centos71_Online_BaseTarball PR
+         validation build onto a machine with Bash.
+      - From the "Artifacts" page, navigate to "Tarball centos71_Online_BaseTarball"
+        and download the `base_tarball_*.tar.gz` and `tarball_*-smoke-test-prereqs.tar.gz`
+        files.
+      - The smoke-test prereqs are platform-independent but should be taken
+        from this non-portable build, *not* CentOS7 Offline Portable.
+      - Can use <https://github.com/dotnet/source-build/blob/release/3.1/scripts/fetch-azdo-artifact-tarball.sh>
+        as a library.
+        - E.g. download the tarballs with `download_tarball`, and fix them up
+          with `fix_azdo_tarball` if necessary.
+1. - [ ] Rename tarball and smoke-test prereqs to human-friendly names.
```

Comment:
Specify the expected human-friendly names.

Reply:
Added.